hardware_interface/CMakeLists.txt

```diff
@@ -27,6 +28,7 @@ target_include_directories(
   include)
 ament_target_dependencies(
   hardware_interface
+  "rcpputils"
```

Comment:
why the quotation marks around these ones?

Reply:
That’s basically ROS2 convention and is used throughout that repo as such.